featuretools/entityset/entityset.py

```diff
@@ -710,6 +710,8 @@ def normalize_entity(self, base_entity_id, new_entity_id, index,
 
         transfer_types = {}
         transfer_types[new_index] = type(base_entity[index])
+        for v in additional_variables + copy_variables:
```

Comment:
Is this bug fix for us currently not transferring the type of `additional_variables` and `copy_variables`?